utils/docker/Dockerfile.centos.8

```diff
@@ -62,14 +61,14 @@ RUN if [ -n "$REPO_APPSTREAM" ]; then                           \
 name=created from ${MY_REPO}\n\
 baseurl=${MY_REPO}\n\
 enabled=1\n\
-repo_gpgcheck=1\n\
+repo_gpgcheck=0\n\
```

Comment:
Why does the repo GPG check need to be disabled?  We really ought to try not to do this as this weakens the integrity of what we are downloading from external repositories.

Reply:
All the other repos in all the other Dockerfiles have this, I just don't think we're ever run or tested APSTREAM.